utils/update_checkout.py

```diff
@@ -168,6 +168,11 @@ def update_single_repository(args):
                       "to rebase.\n")
 
             shell.run(["git", "submodule", "update", "--recursive"], echo=True)
+
+            # We also want to update all the tags with remote. By default git doesn't 
+            # do it and so we need to fetch it
+            # https://stackoverflow.com/questions/1204190/does-git-fetch-tags-include-git-fetch
+            shell.run(["git", "fetch", "--tags"], echo=True)
```

Comment:
It'd be more efficient (fewer connections) to do this at the same time as the other fetch.